### ISSUE TITLE
Light scanning broken

### ISSUE THREAD

**Issue by abnd90** [diyhue/diyHue#157]
**Describe the bug**
It looks like #145 broke light scanning. [This commit](https://github.com/diyhue/diyHue/commit/2a8d4b8bda3dec98724d1acf6d87e0c999a9c9d9) added a "raise" in the light scanning code on an exception. This causes scanning to end on the first failure - unparsable json, devices with a self signed ssl cert, etc.

**Logs**
```
diyhue    | Exception in thread Thread-13:
diyhue    | Traceback (most recent call last):
diyhue    |   File "/usr/lib/python3.7/threading.py", line 917, in _bootstrap_inner
diyhue    |     self.run()
diyhue    |   File "/usr/lib/python3.7/threading.py", line 865, in run
diyhue    |     self._target(*self._args, **self._kwargs)
diyhue    |   File "/opt/hue-emulator/HueEmulator3.py", line 636, in scan_for_lights
diyhue    |     device_data = json.loads(response.text)
diyhue    |   File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
diyhue    |     return _default_decoder.decode(s)
diyhue    |   File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
diyhue    |     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
diyhue    |   File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
diyhue    |     raise JSONDecodeError("Expecting value", s, err.value) from None
diyhue    | json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

```

**Comment by zwegner** [comment]
You're right, thanks, looks like I messed that up. I think I had that commented out, but it snuck into a commit. It's safe to delete that line.

I have that fixed in [our version](https://github.com/atx-led-open-source/diyHue) that I haven't yet made a pull request for yet. All the more reason to get a good test suite running...

**Comment by abnd90** [comment]
I removed that line in my fork and built a docker container. Using that
works well.

+1 for a test suite, I'd even want to contribute code. I'm not very pleased
on having my house lights depend on untested code. Couple of times I
updated to new code, I encountered breakage.

**Comment by JDF1** [comment]
Hello everyone,

Today I have updated my DiyHue Bridge with the easy install curl -s command and I updated one of my my DiyHue lights, with the latest Generic_WS2812_Strip sketch. I have also the issue that there are no DIYHue lights found when I search for lights in the Hue app. 

Is this problem still alive in the easy install version (curl -s .....)

Thanks allot in advanced.

**Comment by alexyao2015** [comment]
Yep I have this exact same problem. Is a commit going to get merged?

**Comment by zamzon** [comment]
Hi

I am new to diyhue, I have bin using Phoscon, I have connected diyhue to Deconz using the hue bridge function,
But how do a add more lights to diyhue? 

when I search in the hue app I don’t find any lights, then I did a search in phoscon and find my new lights, but they didn’t show in the diyhue and the hue app.
The only way I could get them in to the hue app was to pair the diyhue to the phoscon again.

It this the same problem you are talking about?
and it should it automatically pop up new lights in the diyhue/hue app when I press search button in the hue app ?
